core/src/main/java/org/aerogear/mobile/core/metrics/MetricsPublisherListener.java

```diff
@@ -0,0 +1,11 @@
+package org.aerogear.mobile.core.metrics;
+
+
+import org.aerogear.mobile.core.http.HttpResponse;
+
+public interface MetricsPublisherListener {
```

Comment:
1. It's not a listener it's more a callback
2. Why we need a specific one for metrics?

Reply:
1. Listeners are Android's callbacks, like in UI components, it follows the same pattern.
2. The purpose is for the user to handle the outcome of the metrics service actions.

Reply:
You are using the Listener like a callback in the metrics publisher which is causing the confusion.